src/main/groovy/de/dkfz/roddy/tools/TimeUnit.groovy

```diff
@@ -151,4 +151,24 @@ class TimeUnit {
     String toString() {
         return timeString
     }
+
+    String alignToHours() {
+        String[] split = timeString.split("[:]")
```

Comment:
Is there no time parsing and arithmetics code already in the JDK?

Reply:
This is not a parser method. This just reduces the "size2 of a pbs/sge walltime / timeunit string. E.g. 10:12:00:00 => 252:00:00. 10 days => 240 hours.